tests/functional_tests/t0_main/yolov8/test_yolov8_classification.py

```diff
@@ -1,16 +0,0 @@
-from ultralytics.yolo.engine.model import YOLO
```

Comment:
Dont you need to delete the corresponding .yea files in this directory, otherwise the tests will fail.